packages/engine/src/scene/components/ColliderComponent.ts

```diff
@@ -108,8 +108,9 @@ export const ColliderComponent = defineComponent({
     const isLoadedFromGLTF = useOptionalComponent(entity, GLTFLoadedComponent)
     const groupComponent = useOptionalComponent(entity, GroupComponent)
 
+    const isMeshCollider = [ShapeType.TriMesh, ShapeType.ConvexPolyhedron].includes(colliderComponent.shapeType.value)
```

Comment:
shouldn't we also be checking for ModelComponent here?

Reply:
We don't use ModelComponent at any point, just the GroupComponent

Reply:
We should leave the door open to allow any kind of component to supply the mesh that is used